next/utils/prefetchPageSections.ts

```diff
@@ -37,11 +33,12 @@ export const prefetchPageSections = async (page: PageEntityFragment, locale: str
     await queryClient.prefetchQuery(['InbaTags', locale], () => client.InbaTags({ locale }))
   }
 
-  if (sectionTypes.includes('ComponentSectionsOfficialBoard')) {
-    await queryClient.prefetchQuery(getGinisOfficialBoardQueryKey(''), () =>
-      ginisOfficialBoardFetcher(''),
-    )
-  }
+  // TODO this does not work, throws ERR_CONNECTION_REFUSED
+  // if (sectionTypes.includes('ComponentSectionsOfficialBoard')) {
+  //   await queryClient.prefetchQuery(getGinisOfficialBoardQueryKeyJson(), () =>
+  //     ginisOfficialBoardFetcherJson(),
+  //   )
+  // }
```

Comment:
likely a forgotten comment ? if this is changed to officialBoardListFetcher with correct options passed I'd guess it will work